agenta-backend/agenta_backend/models/db_models.py

```diff
@@ -15,7 +16,7 @@ class UserDB(Model):
     uid: str = Field(default="0", unique=True, index=True)
     username: str = Field(default="agenta")
     email: str = Field(default="demo@agenta.ai", unique=True)
-    organization_id: OrganizationDB = Reference(key_name="org")
+    organizations: List[OrganizationDB]
```

Comment:
This should be optional. By default, a user doesn't isn't going to belong to any organization, no?

```suggestion
    organizations: Optional[List[OrganizationDB]]
```